Chinese/Hei Hei Hei/Hei Hei Hei.md

```diff
@@ -0,0 +1,7 @@
+够了！！大家都安静点！不要再扯孙悟空了，都过来看费玉污讲段子了！！
+段子在此：
+[如果你能抓到我](Chinese/Kong-Shi-foo/)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.